lib/NmfTextTagger.jsm

```diff
@@ -3,25 +3,22 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 "use strict";
 
+const {toksToTfIdfVector} = ChromeUtils.import("resource://activity-stream/lib/TfIdfVectorizer.jsm", {});
```

Comment:
I've noticed a few places where we call this "toksToTfIdfVector" and not "TfIdfVectorizer"

I wonder if it make sense to rename the file to match "toksToTfIdfVector" or is the name change semantically correct, just not immediately obvious why?

Reply:
Will change.